lib/segment/src/index/hnsw_index/hnsw.rs

```diff
@@ -19,6 +21,7 @@ use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Arc;
 
 const HNSW_USE_HEURISTIC: bool = true;
+const BYTES_IN_KB: usize = 1024;
```

Comment:
This constant is declared 4 times. Maybe reuse it? And I propose to rename this constant. We use this constant as a parameter unit size. So we could rename to `STORAGE_UNIT_SIZE` or something else